search/includes/classes/class-search.php

```diff
@@ -1451,7 +1498,29 @@ public function filter__ep_skip_post_meta_sync( $skip_sync, $post, $meta_id, $me
 		return false;
 	}
 
-	public function get_post_meta_allow_list( $post ) {
+	/**
+	 * Adds default values for vip_search_post_meta_allow_list by combining vip-search defaults with jetpack values (to ease migration).
+	 *
+	 * @param {array} $keys Input post meta keys.
+	 * @return {array} Post meta keys enhanced by default values.
+	 */
+	public function filter__vip_search_post_meta_allow_list_defaults( $keys ) {
```

Comment:
@pschoffer I simplified the logic of this function a bit in 7fcde723f

Reply:
Great stuff thanks @nickdaugherty !